format/index.md

```diff
@@ -513,6 +513,39 @@ the following members:
   data represents a resource relationship.
 * Pagination links for the primary data (as described below).
 
+Value of each member of a `links` object can be either a string
+containing the link URL or a "link object", which can contain
+the following members:
+
+* `"href"` - the string containing the link URL or an array containing multiple strings containing the link URLs,
```

Comment:
@tkellen What about the [first option discussed here](https://github.com/json-api/json-api/issues/622#issuecomment-103729996), where `href` isn't an array, but the value at the key in `links` can optionally be?